include/ebpf_result.h

```diff
@@ -103,6 +103,9 @@ extern "C"
 
         // The container can not hold additional elements.
         EBPF_OUT_OF_SPACE,
+
+        // Operation was cancelled.
+        EBPF_CANCELLED, // = 30
```

Comment:
```suggestion
        // Operation was canceled.
        EBPF_CANCELED, // = 30
```
"cancelled" is british spelling. "canceled" is american spelling. See https://www.tckpublishing.com/canceled-vs-cancelled/ and lots of other sources.

Reply:
Following the precedence in ntstatus.h
```
//
// MessageId: STATUS_CANCELLED
//
// MessageText:
//
// The I/O request was canceled.
//
#define STATUS_CANCELLED                 ((NTSTATUS)0xC0000120L)
```

That is double-L in the identifier by single-L in comments. Elsewhere in the code I am using single L.

Reply:
I don't think an error in a windows file should be used as a precedent to match in ebpf code.   You should be able to write ebpf code without even knowing there are NTSTATUS values.

Reply:
We treat NT STATUS_CANCELELD as EBPF_CANCELED and vice versa. I will update.